src/js/control-bar/volume-control/volume-bar.js

```diff
@@ -22,6 +22,7 @@ class VolumeBar extends Slider {
     super(player, options);
     this.on(player, 'volumechange', this.updateARIAAttributes);
     player.ready(Fn.bind(this, this.updateARIAAttributes));
+    player.ready(() => this.on(this.el().parentNode, ['mousedown', 'touchdown'], this.handleMouseDown));
```

Comment:
Can we assume that this is safe for anything that parentNode could be, assuming that volume bar could be used independently of the volume menu?